### PR TITLE
WIP: Allow searching by value in banks

### PR DESCRIPTION
**Pull request by Bram91**
Added searching by value in banks.

- `<` smaller than
- `>` larger than
- ` -` intervals
ie:
`value:<5m` will show all itemstacks below 5 million GP
`value:100k-1b` will show all itemstacks starting at 100k up to 1billion GP

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java [hunk 1]
```diff
@@ -54,7 +57,7 @@
 	private final ConfigManager configManager;
 	private final ItemManager itemManager;
 	private final ClueScrollService clueScrollService;
-
+	private Item[] bankItems;
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java]: Tag manager shouldn't be aware of the itemcontainer items.

Comment (Bram91) [reply runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java]: Relocated the itemcontainer items array to its own class.

File: runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankItemCache.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package net.runelite.client.plugins.banktags;
+
+import net.runelite.api.Item;
+
+public class BankItemCache {
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankItemCache.java]: Remove this class, and just inject client and call client.getItemContainer (or w/e it is named atm) in tag manager

**Comment by Bram91** [comment]
Removed all the logic from tagmanager and placed it in BankTagsPlugin also added `HA:` search

**Pull request closed without merging**